core/app/finders/spree/taxons/find.rb

```diff
@@ -0,0 +1,61 @@
+module Spree
+  module Taxons
+    class Find
+      def initialize(taxons, params)
+        @taxons   = taxons
+
+        @roots    = params[:roots]
+        @ids      = params[:ids].split(',') if params[:ids].present?
+        @taxonomy = params[:taxonomy_id]
+        @name     = params[:name]
+      end
+
+      def call
+        taxons = @taxons
```

Comment:
@bbonislawski 
Please take a look at my previous comments related to this section in regard to this assignment & architecture

Reply:
private `attr_accessor` for @taxons would be probably better way to handle this situation.